### ISSUE TITLE
Name change in bazelbuild/rules_cc 

### ISSUE THREAD

**Issue by ChrisReinke** [deepmind/lab#221]
Hello,

it seems there was a name change of the branch in bazelbuild/rules_cc from "master"  to "main".

Here is a corresponding thread: https://github.com/bazelbuild/rules_cc/issues/107

Thus, when I run the code, it does not work and I get:

```
bazel run :game -- --level_script=tests/empty_room_test --level_setting=logToStdErr=true
Starting local Bazel server and connecting to it...
INFO: Repository rules_cc instantiated at:
  /scratchlocal/creinke/data/study/perception/code/test/deepmind_lab/lab/WORKSPACE:11:13: in <toplevel>
Repository rule http_archive defined at:
  /scratchlocal/creinke/anaconda3/envs/deepmindlab/share/bazel/9ae99997a58f82f417398ebc705bfd25/external/bazel_tools/tools/build_defs/repo/http.bzl:336:31: in <toplevel>
WARNING: Download from https://github.com/bazelbuild/rules_cc/archive/master.zip failed: class com.google.devtools.build.lib.bazel.repository.downloader.UnrecoverableHttpException GET returned 404 Not Found
ERROR: An error occurred during the fetch of repository 'rules_cc':
   Traceback (most recent call last):
	File "/scratchlocal/creinke/anaconda3/envs/deepmindlab/share/bazel/9ae99997a58f82f417398ebc705bfd25/external/bazel_tools/tools/build_defs/repo/http.bzl", line 111, column 45, in _http_archive_impl
		download_info = ctx.download_and_extract(
Error in download_and_extract: java.io.IOException: Error downloading [https://github.com/bazelbuild/rules_cc/archive/master.zip] to /scratchlocal/creinke/anaconda3/envs/deepmindlab/share/bazel/9ae99997a58f82f417398ebc705bfd25/external/rules_cc/temp4497128067373690864/master.zip: GET returned 404 Not Found
ERROR: Error fetching repository: Traceback (most recent call last):
	File "/scratchlocal/creinke/anaconda3/envs/deepmindlab/share/bazel/9ae99997a58f82f417398ebc705bfd25/external/bazel_tools/tools/build_defs/repo/http.bzl", line 111, column 45, in _http_archive_impl
		download_info = ctx.download_and_extract(
Error in download_and_extract: java.io.IOException: Error downloading [https://github.com/bazelbuild/rules_cc/archive/master.zip] to /scratchlocal/creinke/anaconda3/envs/deepmindlab/share/bazel/9ae99997a58f82f417398ebc705bfd25/external/rules_cc/temp4497128067373690864/master.zip: GET returned 404 Not Found
ERROR: While resolving toolchains for target //:game: com.google.devtools.build.lib.packages.RepositoryFetchException: no such package '@rules_cc//cc': java.io.IOException: Error downloading [https://github.com/bazelbuild/rules_cc/archive/master.zip] to /scratchlocal/creinke/anaconda3/envs/deepmindlab/share/bazel/9ae99997a58f82f417398ebc705bfd25/external/rules_cc/temp4497128067373690864/master.zip: GET returned 404 Not Found
ERROR: Analysis of target '//:game' failed; build aborted: com.google.devtools.build.lib.packages.RepositoryFetchException: no such package '@rules_cc//cc': java.io.IOException: Error downloading [https://github.com/bazelbuild/rules_cc/archive/master.zip] to /scratchlocal/creinke/anaconda3/envs/deepmindlab/share/bazel/9ae99997a58f82f417398ebc705bfd25/external/rules_cc/temp4497128067373690864/master.zip: GET returned 404 Not Found
INFO: Elapsed time: 3.532s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (1 packages loaded, 0 targets configured)
FAILED: Build did NOT complete successfully (1 packages loaded, 0 targets configured)
    currently loading: @bazel_tools//tools/cpp
```

It works when I change in the WORKSPACE file lines 11 - 15 to

```
http_archive(
    name = "rules_cc",
    strip_prefix = "rules_cc-main",
    urls = ["https://github.com/bazelbuild/rules_cc/archive/main.zip"],
)
```

**Comment by tkoeppe** [comment]
Yes, we'll have to fix that. I'll send a patch. Thanks!

**Comment by tkoeppe** [comment]
Hm, looks like we don't even need this dependency anymore. It must have been used by something else in the past, but I don't see it now.

**Comment by tkoeppe** [comment]
FIxed by 6e3572574da312c77c1272d0beceb4292ded6fc0.
